src/frontends/tensorflow/src/translate_session.cpp

```diff
@@ -11,6 +11,35 @@
 
 using namespace ov::frontend::tensorflow;
 
+namespace {
+template <typename T>
+T reorder_ops_by_names(const std::vector<std::string>& names, const T& ops) {
```

Comment:
I think this is too broad template, can it be `vector<T>` at least?

Reply:
Done